README.md

```diff
@@ -56,6 +56,12 @@ The application under ./SwipeListExample will produce the above example. To run
 ```javascript
 import { SwipeListView } from 'react-native-swipe-list-view';
 
+//... note: your data array objects MUST contain a key property to ensure proper functionality
```

Comment:
Can you amend this to say "MUST contain a key property OR a `keyExtractor` prop..."

Reply:
I'm trying to look into the source to see what the usage of `keyExtractor` is

I'm seeing it referenced in `SwipeListView.js` twice as part of the props. However, I don't see it listed in the `propTypes`
I see it being called as `keyExtractor(item, index)` so I can add this to the readme as well as the proptypes if that seems correct